infra/ansible/remote/nodes/playbooks/start/metagraph-l0/genesis.ansible.yml

```diff
@@ -293,44 +293,42 @@
     start_args: "{{ 'genesis.snapshot --metagraph-owner-message ./owner-message' if should_run_genesis else '' }}"
     node_environment: "{{ genesis_env if should_run_genesis else rollback_env }}"
 
-# Wait for node to be online and send messages
-- name: Wait for node to be online
+# Wait for node to be online and Ready before sending messages
+- name: Wait for node to be Ready
   uri:
     url: "http://localhost:{{ base_metagraph_l0_public_port }}/node/info"
     method: GET
+    return_content: yes
   register: node_status
-  until: node_status.status == 200
+  until: node_status.status == 200 and ("Ready" in node_status.content)
   retries: 120
-  delay: 1
+  delay: 2
 
 - name: Wait before sending messages
   pause:
-    seconds: 10
-  when: 
-    - owner_p12_file_name is defined and owner_p12_file_name != ""
-    - should_run_genesis or force_owner_message
+    seconds: 15
 
-- name: Send staking message
-  uri:
-    url: "http://localhost:{{ base_metagraph_l0_public_port }}/currency/message"
-    method: POST
-    body: "{{ staking_message }}"
-    body_format: json
-    status_code: [200, 204]
-  when: 
+- name: Send staking message via curl
```

Comment:
I had a lot of trouble getting this to work with Ansible 2.20's uri module, so I opted to change it for simple curl instead, which works smoother in my experience and also waiting for the node to be ready first